sql/pgwire_test.go

```diff
@@ -765,13 +765,13 @@ func TestPGPreparedExec(t *testing.T) {
 			},
 		},
 		{
-			"CREATE TABLE d.types (i int, f float, s string, b bytes, d date, m timestamp, n interval, o bool, e decimal)",
+			"CREATE TABLE d.types (i int, f float, s string, b bytes, d date, m timestamp, z timestamp with time zone, n interval, o bool, e decimal)",
 			[]preparedExecTest{
 				base,
 			},
 		},
 		{
-			"INSERT INTO d.types VALUES ($1, $2, $3, $4, $5, $6, $7, $8, $9)",
+			"INSERT INTO d.types VALUES ($1, $2, $3, $4, $5, $6, $7::timestamptz, $8, $9, $10)",
```

Comment:
I'm a little bummed about this cast... for some reason I thought we were able to resolve the desired type here without it, but I guess we're missing the info when we type the placeholders?